### PR TITLE
Force skinny nav on settings pages

### PR DESCRIPTION
**Pull request by Lanny**
## Description

This is how it was intended to work before, but in the process of fixing the nav when pre-rendering I broke this. This isn't a super pretty kludge that involves rolling NavLinks into Nav, but I can't think of a better way to handle it. We could make it slightly cleaner by using a css preprocessor (SASS, LESS, whatever, it doesn't really matter) with `styled-jsx` but I don't really want to bring in the whole dep just for this.

## Migrations

No migs

### CODE REVIEW

File: components/Dashboard/Nav/Nav.tsx [hunk 1]
```diff
@@ -32,14 +49,77 @@ const Nav: React.FC<Props> = ({ expanded, collapse }) => {
...
+    handleCollapse()
+    logout()
+
+    // Clear the currentUser cache
+    refetch()
+
+    // Redirect to home page
+    Router.push('/')
+  }
+
```
Comment (robin-macpherson) [inline components/Dashboard/Nav/Nav.tsx:62]: **NICE** thanks for fixing this!!

**Review by robin-macpherson — APPROVED** [review]
### WOOOOOOOOO THANKS FOR FIXING THESE! 💃🏻 🦔 💃🏻

File: components/Dashboard/Nav/Nav.tsx (hunk 1 above)
Comment (robin-macpherson) [reply components/Dashboard/Nav/Nav.tsx:62]: Ah, @Lanny I just tested logout on staging and I'm seeing the same behavior as before. Super happy to get the nav fixed on settings page but just checking if this code was intended to also fix logout?

Comment (Lanny) [reply components/Dashboard/Nav/Nav.tsx:62]: It was not, this is probably the piece of code to look at but this PR was just trying to get the nav working on the settings page.

Comment (robin-macpherson) [reply components/Dashboard/Nav/Nav.tsx:62]: Ah ok cool will merge now then! 😄

**Pull request merged**